src/ui/atoms/list/List.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import type { DeepReadonly } from 'superTypes'
+import './list.scss'
+import type { ListItemProps } from '../listItem/ListItem'
+import { ListItem } from '../listItem/ListItem'
+
+export type ListProps = {
+  /**
+   * The items of the list.
+   */
+  readonly items: ListItemProps[]
+  /**
+   * An icon displayed on the left side of each items.
+   */
+  readonly leftIcon?: JSX.Element
+  /**
+   * An icon displayed on the right side of each items.
+   */
+  readonly rightIcon?: JSX.Element
```

Comment:
I can't see clearly here the purpose of the right and left icons 🤔
IMHO, I think these properties should be only the responsibility of the `ListItem` because the `List` only maps and renders child items, it shouldn't influence the content of an element.
Furthermore, it seems to be redundant with `ListItem` properties.